applications/vanilla/modules/class.tagmodule.php

```diff
@@ -102,6 +102,18 @@ public function getData() {
         $tagQuery = Gdn::sql();
 
         $this->autoContext();
+        //Allow addon to manipulate the data being rendered
+        $tagData = null;
+        $this->EventArguments['ParentID'] = $this->ParentID;
+        $this->EventArguments['ParentID'] = $this->ParentType;
+        $this->EventArguments['tagData'] = &$tagData;
+        $this->fireEvent('getData');
+        if ($tagData !== null) {
+            if (is_array($tagData)) {
```

Comment:
At this point, since we're checking if it's not null, then checking if it's an array, couldn't we just check to see if it's an array (i.e. drop the is-null condition and bring out the `is_array` check)? If there's a reason to keep the null check, couldn't you combine the condition from the inner block?

Reply:
I think we drop the is_null check, but I don't think it's necessary